modules/actions/workflows.go

```diff
@@ -44,6 +44,19 @@ func ListWorkflows(commit *git.Commit) (git.Entries, error) {
 	return ret, nil
 }
 
+func converGithubEvent2GiteaEvent(evt *jobparser.Event) string {
+	switch evt.Name {
+	case "pull_request", "pull_request_target", "pull_request_review_comment", "pull_request_review":
+		return string(webhook_module.HookEventPullRequest)
+	case "registry_package":
+		return string(webhook_module.HookEventPackage)
+	case "create", "delete", "fork", "push", "issues", "issue_comment", "release", "pull_request_comment":
+		fallthrough
+	default:
+		return evt.Name
+	}
+}
+
```

Comment:
I would not write a converter Function here, I would write it in the `jobparser` package and make it publicly available. Also, this would pop out when you use Intellisense in the suggestion windows and is reusable
  
```suggestion
func (evt *jobparser.Event) ConverGithubEvent2GiteaEvent string {
	switch evt.Name {
	case "pull_request", "pull_request_target", "pull_request_review_comment", "pull_request_review":
		return string(webhook_module.HookEventPullRequest)
	case "registry_package":
		return string(webhook_module.HookEventPackage)
	case "create", "delete", "fork", "push", "issues", "issue_comment", "release", "pull_request_comment":
		fallthrough
	default:
		return evt.Name
	}
}

```

Reply:
I don't think so. Because we can support to convert other sub-system events to Gitea Event. i.e. from GitLab's